### PR TITLE
Kubernetes Response Engine, Part 4: Tekton

### PR DESCRIPTION
**Pull request by NissesSenap**
**What type of PR is this?**

/kind content

**Any specific area of the project related to this PR?**

/area blog

**What this PR does / why we need it**:
Showcase Tekton and falco demo building a Kubernetes Response Engine

### CODE REVIEW

**Comment by NissesSenap** [comment]
I'm going to update the link in the header to point on https://github.com/falcosecurity/falco-website/pull/425 but I knew that the CI system would fail due to broken link if i didn't. So when that one is merged I will update this one.

File: content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+2021/05/02 18:11:00 PodName: alpine & Namespace: falcoresponse
+2021/05/02 18:11:00 Rule: Terminal shell in container
+2021/05/02 18:11:00 Deleting pod alpine from namespace falcoresponse
+```
+
+## Conclusion
+
+This was a rather simple example on how we can use the power of tekton together with Falco to protect us from bad actors that is trying to take over pods in our cluster.
+
+As noted during this post there are allot of potential improvements before this is production ready:
```
Comment (phillebaba) [inline content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md:605]: a lot

File: content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md [hunk 2]
```diff
@@ -0,0 +1,613 @@
...
+author: Edvin Norling
+slug: falcosidekick-reponse-engine-part-4-tekton
+---
+
+> *This blog post is part of a series of articles about how to create a `Kubernetes` response engine with `Falco`, `Falcosidekick` and a `FaaS`.*
+>
+> See other posts:
+>
+> - [Kubernetes Response Engine, Part 1 : Falcosidekick + Kubeless]({{< ref "/blog/falcosidekick-reponse-engine-part-1-kubeless" >}})
+> - [Kubernetes Response Engine, Part 2 : Falcosidekick + OpenFaas]({{< ref "/blog/falcosidekick-reponse-engine-part-2-openfass" >}})
```
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md:13]:
```suggestion
> - [Kubernetes Response Engine, Part 2 : Falcosidekick + OpenFaas]({{< ref "/blog/falcosidekick-reponse-engine-part-2-openfass" >}})
> - [Kubernetes Response Engine, Part 3 : Falcosidekick + Knative]({{< ref "/blog/falcosidekick-reponse-engine-part-3-knative" >}})
```

**Review by Issif — CHANGES_REQUESTED** [review]

**Comment by NissesSenap** [comment]
Updated the typos and linked back to part 3

File: content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md (hunk 2 above)
Comment (NissesSenap) [reply content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md:13]: Didn't see that you fixed it as well, so thats why I didn't use yours.
Thanks

File: content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md [hunk 3]
```diff
@@ -0,0 +1,614 @@
+---
+title: "Kubernetes Response Engine, Part 4: Falcosidekick + Tekton"
+date: 2021-05-10
+author: Edvin Norling
+slug: falcosidekick-reponse-engine-part-4-tekton
```
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md:5]:
```suggestion
slug: falcosidekick-response-engine-part-4-tekton
```

File: content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md [hunk 4]
```diff
@@ -0,0 +1,614 @@
+---
+title: "Kubernetes Response Engine, Part 4: Falcosidekick + Tekton"
+date: 2021-05-10
```
Comment (Issif) [inline content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md:3]: I think, you should update this date after having fixed my suggestion

File: content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md [hunk 5]
```diff
@@ -0,0 +1,612 @@
...
+This was a rather simple example on how we can use the power of tekton together with Falco to protect us from bad actors that is trying to take over pods in our cluster.
+
+As noted during this post there are allot of potential improvements before this is production ready:
+
+- The criticalNamepsaces in our go code is currently hard-coded and needs to be input variable of some kind.
+- We need to be able to delete pods depending on priority level, rule or something similar.
+- To be able to debug pods we might need to shell in to them, we need a way to ignore pods temporary without the pod getting restarted. Probably a annotation to look for in the pod before deleting it.
+- And probably many other needs that you can come up with.
+
+If you have any ideas/issues come and share them in the falco slack [https://kubernetes.slack.com](https://kubernetes.slack.com) #falco.
```
Comment (danpopSD) [inline content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md:614]: add this 

Falco
If you would like to find out more about Falco:

Get started in Falco.org.
Check out the Falco project in GitHub
Get involved in the Falco community.
Meet the maintainers on the Falco Slack.
Follow @falco_org on Twitter.

as the other blogs in this series https://falco.org/blog/falcosidekick-reponse-engine-part-3-knative/

perhaps @NissesSenap / @Issif   we should also put details on Tekton (for more information?)

**Review by danpopSD — COMMENTED** [review]
great work i think it needs a little finesse at the end. to show how more people can contribute

File: content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md (hunk 5 above)
Comment (NissesSenap) [reply content/en/blog/falcosidekick-reponse-engine-part-4-tekton.md:614]: Done, also linked back to tekton

**Review by Issif — APPROVED** [review]

**Pull request merged**